bin/resin-preload

```diff
@@ -160,6 +160,7 @@ const spinnerHandler = (event) => {
   if (event.action === 'start') {
     spinner.start()
   } else {
+    console.log()
```

Comment:
What is this for?

Reply:
It's the same as https://github.com/resin-io/resin-cli/pull/687#discussion_r144904431

Reply:
Ah okay – can you also add a short comment about that here?

Reply:
Added